src/adapt/cssparse.js

```diff
@@ -336,6 +336,19 @@ adapt.cssparse.ParserHandler.prototype.endRule = function() {
 };
 
 /**
+ * @return {void}
+ */
+adapt.cssparse.ParserHandler.prototype.startNotRule = function() {
```

Comment:
The name `startNotRule` looks weird since `:not()` is not a rule.
Considering that there are other functional pseudo-classes taking a selector list as an argument (e.g. [`:matches()`](https://drafts.csswg.org/selectors-4/#matches) or [`:has()`](https://drafts.csswg.org/selectors-4/#relational)), this handler method should not be tied to `:not()`.
I imagine something like this:
```javascript
/**
 * @param {string} funcName The name of the function taking a selector list as an argument
 */
adapt.cssparse.ParserHandler.prototype.startFuncWithSelector = function(funcName) {}
```